l1-contracts/src/core/Decoder.sol

```diff
@@ -92,21 +103,27 @@ contract Decoder {
    * @return sha256(header[0x4: 0x1ac], diffRoot)
```

Comment:
The comment should be added as well with the sice, `0x23c` and the `l1ToL2messagesHash`